kroxylicious-operator/src/main/resources/kubernetes/Crd-KafkaProxy.yaml

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
There is no `-v1` in the filename, is either option a deliberate choice (I know I'm really hitting on the big issues today :sigh:)

Reply:
TBH I find it confusing to include any `v` in the file name. If we did, it must be the `v` for the `CustomResourceDefinition` API, rather than one of the `v`s for the API we're declaring, right? Because a single CRD is used to declare all versions of an API, so what ought the file name be if the CRD in it is declaring multiple versions?

That being the case, it's not clear to me why we wouldn't also want to include the group of the `CustomResourceDefinition` API in the file name too. That would at least make it less ambiguous what the `v` in the file name was referring to. 

But from a pragmatic PoV: 

* We want to avoid having multiple files floating around declaring the same CRD (otherwise there's the risk of confusion about which is the source of truth).
* If we're going to stick with the `LocallyRunOperatorExtension` (and while the bug I opened exists) then we have to Use the file naming which it is forcing on us.
* Alternatively (i.e. the `LocallyRunOperatorExtension` behaviour is intentional), we could decide that `LocallyRunOperatorExtension` isn't giving us much value (what with it also needing a running kube), and decide just not to use it, and get test coverage another way.